framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -361,7 +361,7 @@ void MediaPlayerImpl::pausePlayer()
 
 int MediaPlayerImpl::getVolume()
```

Comment:
i think the return type of getVolume funtion seems to be appropriate for unsigned int.

Reply:
I think so, too.
But let's modify this in another PR, I'll mark ToDo here.